src/zenml/client.py

```diff
@@ -3262,68 +3261,114 @@ def delete_trigger(
 
     def get_snapshot(
         self,
-        id_or_prefix: Union[str, UUID],
+        name_id_or_prefix: Union[str, UUID],
+        *,
+        pipeline_name_or_id: Optional[Union[str, UUID]] = None,
         project: Optional[Union[str, UUID]] = None,
+        include_config_schema: Optional[bool] = None,
+        allow_prefix_match: bool = True,
```

Comment:
Just FYI introducing this with True default value changes the existing default behavior for all `get_snapshot` invocations throughout all existing code, because previously a fixed match would've been expected.

I checked all calls and, with a couple of exceptions, it's used only with UUIDs.